daniel-davidF/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Lab 1</title>
+  <!-- DONE: Add a meta tag to properly set the viewport and initial scale -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="vendor/styles/normalize.css">
+  <!-- REVIEW: Icon Fonts - Lets talk about why and how to use them best! -->
+  <link rel="stylesheet" href="styles/icons.css">
+  <link rel="stylesheet" href="styles/base.css">
+  <link rel="stylesheet" href="styles/layout.css">
+  <link rel="stylesheet" href="styles/modules.css">
+  <link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200">
+</head>
+<body>
+  <header class="site-header clearfix">
+    <section class="heading-group-h1">
+      <h1>
+        Kilovolt Blog!
+      </h1>
+      <h2>The shocking power of static pages</h2>
+    </section>
+    <nav class="main-nav">
+      <!-- DONE: Add a hamburger menu icon here. -->
+      <img class="icon-menu" src="if_menu-alt_134216 (1).png" alt="">
```

Comment:
You do not need to have an image tag with your icon, since you have the Icomoon font files locally.